GENERATORS/SIMULATIONS/WAVEPACKET/WAVEPACKET_test_.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+def test_WAVEPACKET(mock_flojoy_decorator):
+    import WAVEPACKET
+    from flojoy import DefaultParams
+
+    default = DefaultParams(
+        node_id="WAVEPACKET", job_id="0", jobset_id="0", node_type="default"
+    )
+
+    # node under test
+    res = WAVEPACKET.WAVEPACKET(default_params=default)
```

Comment:
It would be nice to test the expected behaviour of this node a bit more than just the length and type of the output.

From reading the code, this seems like a function that runs over multiple timesteps, can we maybe run it in a loop, inspect the output over time and see if it has the properties we expect it to have ? (e.g. a Gaussian bell curve)